.gitmodules

```diff
@@ -16,3 +16,6 @@
 [submodule "ext_libs/eigen"]
 	path = ext_libs/eigen
 	url = https://gitlab.com/libeigen/eigen.git
+[submodule "ext_libs/vcpkg"]
+	path = ext_libs/vcpkg
+	url = ../../microsoft/vcpkg.git
```

Comment:
we should use the whole url instead of a relative path for it

Reply:
I think relative path resolves https vs ssh based on your clone